pkg/audit/audit_test.go

```diff
@@ -45,11 +48,54 @@ func (s *testAuditSuite) TestLabelMatcher(c *C) {
 	c.Assert(matcher.Match(labels2), Equals, false)
 }
 
+// TestPrometheusHistogramBackend is used to test result manually
+func (s *testAuditSuite) TestPrometheusHistogramBackend(c *C) {
```

Comment:
ditto

Reply:
fix it by adding a HTTP server and handling promhttp